parsers/camelcasify/README.md

```diff
@@ -18,33 +20,73 @@ interface parser {
 
 ### Options
 
-| parameter              | Require    | type      | default    | description                                                                                                                |
+| Parameter              | Required   | Type      | Default    | Description                                                                                                                |
 | ---------------------- | ---------- | --------- | ---------- | -------------------------------------------------------------------------------------------------------------------------- |
-| `keys`                 | optional   | `Array`   | `["name"]` | the list of keys where the function will be applied                                                                        |
-| `excludeFileExtension` | optional   | `Boolean` | `false`    | the fact that the function will only be applied on a filename. Useful for transforming strings containing a file extension |
+| `keys`                 | false      | `Array`   | `["name"]` | The list of keys where the function will be applied                                                                        |
+| `excludeFileExtension` | false      | `Boolean` | `false`    | The fact that the function will only be applied on a filename. Useful for transforming strings containing a file extension |
 
-## Example 
+## Usage
+### Config in `.specifyrc.json`
 
 ```json
 {
-    "name": "camelcasify",
-    "options": {
-      "keys": ["name"]
+  "parsers": [
```

Comment:
If you write the key `parsers` you are in the rule's context. But here it's specific parser documentation.